kernelci/db/kernelci_api.py

```diff
@@ -57,6 +57,17 @@ def _put(self, path, data=None):
         resp.raise_for_status()
         return resp
 
+    def create_user(self, username, password, is_admin):
+        """Create new user"""
+        path = '/'.join(['user', username])
+        if is_admin:
+            path = '?'.join([path, 'is_admin='+str(is_admin)])
+
+        data = {'password': password}
+        resp = self._post(path, json.dumps(data))
+        print(resp.json())
```

Comment:
Please drop this, the calling code can decide to print some information if needed.